Makefile

```diff
@@ -7,6 +7,7 @@ VERSION ?= $(shell git describe --tags --dirty  --always)
 GOOS ?= $(shell ./bin/go version | awk '{print $$NF}' | cut -d/ -f1)
 GOARCH ?= $(shell ./bin/go version | awk '{print $$NF}' | cut -d/ -f2)
 BIN = $(BUILD_DIR)/hermit-$(GOOS)-$(GOARCH)
+export HERMIT_EXE =
```

Comment:
Why do we need to to do this empty export?

Reply:
Ah that's because when using HERMIT_EXE for development we don't want it to be used by the Makefile itself.